src/main/java/uk/gov/hmcts/ccd/domain/model/std/CaseAssignedUserRoleRequest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.ccd.domain.model.std;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@ToString
+@NoArgsConstructor
+@Getter
+public class CaseAssignedUserRoleRequest extends CaseAssignedUserRole {
```

Comment:
The 2 class names don't have a parallelism in the semantics, breaching the expected connotation of an IS-A relation. The sentence `CaseAssignedUserRoleRequest` IS A `CaseAssignedUserRole` is technically correct, but linguistically wrong. Could you please propose a better fitting name for one or both of these to have a proper IS-A reading?

Reply:
As discussed: will rename as `CaseAssignedUserRoleWithOrganisation`